Pending_updates/cit_029_municipal_waste/README.md

```diff
@@ -0,0 +1,16 @@
+## Municipal Waste Dataset Pre-processing
+This file describes the data pre-processing that was done to [the Municipal Waste Generated per Capita](https://stats.oecd.org/Index.aspx?DataSetCode=MUNW) for [display on Resource Watch](https://resourcewatch.org/data/explore/10337db6-8321-445e-a60b-28fc1e114f29).
+
+The source provided the data as a csv file.
+
+Below, we describe the steps used to reformat the table so that it is formatted correctly to upload to Carto.
+
+1. Read in the data as a pandas dataframe and remove the duplicated column 'YEA'. 
```

Comment:
You only need to use a list format if there are a series of steps that are asier to read as a list. If the pocessing is really short, you can just write in paragraph form.
```suggestion
In order to process this data for display on Resource Watch, the data was read into python as a pandas dataframe. The columns 'YEA' and 'Year' are identical, so the 'YEA' column was removed. 
```